src/promptflow/tests/sdk_cli_test/e2etests/test_trace.py

```diff
@@ -247,6 +247,12 @@ def test_delete_traces_with_collection_and_started_before(self, pf: PFClient) ->
         pf.traces.delete(collection=collection2, started_before=delete_query_time.isoformat())
         assert len(pf.traces.list_line_runs(collection=collection2)) == 0
 
+    def test_delete_traces_dry_run(self, pf: PFClient) -> None:
+        mock_run = str(uuid.uuid4())
```

Comment:
add more test for exceptions?

Reply:
have added some in previous delete PR, in test `test_validate_delete_query_params`